src/item_factory.h

```diff
@@ -3,6 +3,7 @@
 
 #include "json.h"
 #include "iuse.h"
+#include "bodypart.h"
```

Comment:
Where? Not in this file as far as I can tell.  If it's used elsewhere include bodypart.h there.